zbus/src/object_server/mod.rs

```diff
@@ -18,10 +18,11 @@ use zvariant::{ObjectPath, OwnedObjectPath, OwnedValue, Signature, Type, Value};
 
 use crate::{
     async_lock::{RwLock, RwLockReadGuard, RwLockWriteGuard},
+    connection::WeakConnection,
```

Comment:
maybe this should just be `connection::Weak`, similar to `std::rc::Weak` and `std::sync::Weak`?

Reply:
yes, I thought about it and forgot, let's do it

Reply:
nah that conflicts with other Weak types.. it's internal anyway, not a big deal.